plugins/forc-index/src/ops/forc_index_start.rs

```diff
@@ -132,7 +137,7 @@ pub async fn init(command: StartCommand) -> anyhow::Result<()> {
             child.id()
         );
     } else {
-        anyhow::bail!("❌ Failed to spawn fuel-indexer child process.");
+        panic!("❌ Failed to spawn fuel-indexer child process.");
```

Comment:
Was this supposed to be changed from `anyhow::bail!` to `panic!`? If so, why?

Reply:
@deekerno 
- Yea. If you run `forc index start` when `fuel-indexer` is not in the path it fails, but the error gets swallowed  by `anyhow::bail` (the process will die and you'll get nothing in stdout)
- I think we can`panic` here (probably elsewhere) where we don't need to do anything after the error (though I definitely should include the error in the message)